internal/hammer/hammer.go

```diff
@@ -364,7 +364,7 @@ func (a *HammerAnalyser) errorLoop(ctx context.Context) {
 // dupChance provides the probability that a new leaf will be a duplicate of a previous entry.
 // Leaves will be unique if dupChance is 0, and if set to 1 then all values will be duplicates.
 // startSize should be set to the initial size of the log so that repeated runs of the
-// hammer can start seeding leaves to avoid duplicates with previous runs.
+// hammer can start seeding leaves to avoid duplicates with p:e revious runs.
```

Comment:
Do you use vim, BTW?

Reply:
This is referring to duplicate leaves showing a profit-to-earnings ratio resulting in _envious revenue_ (or "p:e revious", if you will).

*cough*

fixed.